hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/IncrementalTableBackupClient.java

```diff
@@ -343,6 +350,7 @@ protected void incrementalCopyHFiles(String[] files, String backupDest) throws I
     try {
       LOG.debug("Incremental copy HFiles is starting. dest=" + backupDest);
       // set overall backup phase: incremental_copy
+      // TODO - This should now happen elsewhere maybe
```

Comment:
can you elaborate on this idea? preferably in the comment where it sits in code.

Reply:
ah this was leftover from some other work, I had forgotten to clean this up, thanks for the comment